docs/react-native/features/tracing.md

```diff
@@ -1,14 +1,13 @@
 ---
-title: Performance Tracing
-description: Record traces to monitor the production performance and success rates of operations within your mobile app.
+title: Performance Tracing 
+description: Record spans to monitor the production performance and success rates of operations within your mobile app.
 sidebar_position: 3
 ---
-
 # Performance Tracing
 
 ## Overview
 
-Embrace’s Performance Tracing solution gives you visibility into any app operation you’d like to track, including duration, success rate, and any contextual metadata collected at runtime that helps debug the root cause of your mobile app's performance issues. With our tool, you can quickly spot any bottlenecks in your app’s architecture, pinpoint areas you need to troubleshoot with high precision, and ultimately deliver a truly optimized user experience.
+Embrace’s Traces solution gives you visibility into any app operation you’d like to track, including duration, success rate, and any contextual metadata collected at runtime that helps debug the root cause of your mobile app's performance issues. With our tool, you can quickly spot any bottlenecks in your app’s architecture, pinpoint areas you need to troubleshoot with high precision, and ultimately deliver a truly optimized user experience.
```

Comment:
How broadly are we applying this rename? E.g. the header and title for this page is still "Performance Tracing", should that be changed as well?

Reply:
Updated